src/renderer/components/dialogs/Settings.js

```diff
@@ -114,7 +114,8 @@ class Settings extends React.Component {
               <Login
                 addr={deltachat.credentials.addr}
                 onSubmit={this.onLoginSubmit}
-                loading={deltachat.configuring}>
+                loading={deltachat.configuring}
+                addrDisabled>
```

Comment:
Aha, is this the same thing as doing `addrDisabled={addrDisabled}`?

Reply:
no this is the same thing as `addrDisabled={true}`, its annoying but think of it like HTML dom properties